ops-bedrock/overrides/sepolia/docker-compose.override.yml

```diff
@@ -0,0 +1,53 @@
+services:
+  op-node:
+    depends_on:
+     - op_stack_go_builder
+     - l2
+    command: >
+      op-node
+      --l1=${L1_RPC_URL_WSS}
+      --l2=http://l2:8551
+      --l2.jwt-secret=/config/test-jwt-secret.txt
+      --sequencer.enabled
+      --sequencer.l1-confs=0
+      --verifier.l1-confs=0
+      --p2p.sequencer.key=8b3a350cf5c34c9194ca85829a2df0ec3153be0318b5e2d3348e872092edffba
```

Comment:
in general should also be secret. also is it possible that we are using the default devnet key here?
maybe that's the reason for the peer connections in the op-node, other peers connect to us due to the known address

Reply:
Yes we are using the devnet key here. Of course, and as done with other sensitive information in this PR, as soon as we change it to private information we need to turn this into ENV variables.